k8s/openebs-pre-release-features.yaml

```diff
@@ -1161,28 +1161,18 @@ metadata:
   name: jiva-volume-create-default-0.7.0
 spec:
   defaultConfig:
-  - name: VolumeMonitor
-    enabled: "true"
   - name: ControllerImage
     value: openebs/jiva:0.6.0
   - name: ReplicaImage
     value: openebs/jiva:0.6.0
+  - name: VolumeMonitorImage:ci
```

Comment:
Invalid name

Reply:
thanks for catching this @AmitKumarDas. Fixed it.